tidb-cloud/changefeed-sink-to-cloud-storage.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Sink to Cloud Storage
+Summary: Learn how to create a changefeed to stream data from a TiDB Dedicated cluster to cloud storage, such as Amazon S3 etc.
+---
+
+# Sink to Cloud Storage
+
+This document describes how to stream data from a TiDB Dedicated cluster to cloud storage, such as Amazon S3 etc.
+
+
+> ## Limitations
+> 
+> - To use the Changefeed feature, please ensure that your TiDB cluster is running version v6.5.0 or a later version.
```

Comment:
Sync CDC team, we only support sink to S3 on cloud with version >= 7.1.0 since data folder structure has incompatible changes to lower versions.